visualization/app/codeCharta/util/deltaGenerator.ts

```diff
@@ -89,7 +109,8 @@ export class DeltaGenerator {
 				deltaAttribute[key] = -referenceAttribute[key]
 			}
 		}
-		return deltaAttribute
+
+		return { ...deltaAttribute, addedFiles: 0, removedFiles: 0 }
```

Comment:
Instead of copying the deltaAttributes here, it's possible to set these up front in [line 98](https://github.com/MaibornWolff/codecharta/pull/2701/files#diff-5a4760ef17d74c30e6235c456ecbf0bf5754ff461c369b3506cd5fb1e294f04fR98).

Please be aware that adding values like these mean that it's not possible to use the same names as metric name and we do not control what metric names are loaded into CodeCharta. It's rather unlikely that anyone uses these names and therefore it's fine to implement it like this for now.